azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -535,7 +539,7 @@ V1PodSpec createPodSpec(final int executionId, final VersionSet versionSet,
     final String flowContainerCPURequest = getFlowContainerCPURequest(flowParam);
     final String flowContainerMemoryRequest = getFlowContainerMemoryRequest(flowParam);
     final AzKubernetesV1SpecBuilder v1SpecBuilder =
-        new AzKubernetesV1SpecBuilder(this.clusterName, Optional.empty())
+        new AzKubernetesV1SpecBuilder(this.clusterEnv, Optional.empty())
```

Comment:
Please check if a unit test output needs to be updated for this.

Reply:
Out unit test relies on the variable name passed to the spec builder rather than calling the KubernetesContainerizedImpl. So the unit tests are ok with this.